src/rules/props-order.ts

```diff
@@ -83,7 +82,6 @@ export const propsOrderRule: TSESLint.RuleModule<"invalidOrder", Options> = {
         };
         const sorted = sortAttributes(node.attributes, config);
 
-        const sourceCode = getSourceCode();
```

Comment:
Maybe...

```ts
const sourceCode = getSourceCode() ?? sourceCode ?? undefined;
if (!sourceCode) {
  return;
}
```

Reply:
Until now, getSourceCode existed or did not exist depending on the version of typescript-eslint on the user's side, but with flat config, the user will always use eslint v9, so calling sourceCode as is is no problem here. 
https://eslint.org/docs/latest/use/migrate-to-9.0.0#removed-sourcecode-getcomments

In other words, there is no problem with the current code.